src/docs/asciidoc/en/firebirddocs/nbackup/firebird-nbackup.adoc

```diff
@@ -977,8 +1016,13 @@ In second table: self-restore -> user restore.
 |27 Jun 2020
 |MR
 |Conversion to AsciiDoc, minor copy-editing
-|===
 
+
+|1.6
```

Comment:
Also update version number and date in header of document, and add yourself to the author list.

Also add yourself to the contributors list in the _License notice_ appendix